WebKit build fails - icu cannot build as MSVC fails to compile a test program

Can't build using the latest Docker container. Fails in both `webkitdev/buildbot-worker:windows-2022` or `webkitdev/msbuild-2022:windows-2022` - the MSVC cl.exe fails when CMake tests it to compile a simple program.

<details><summary>Output from build-webkit --release</summary>
<pre>
PS C:\webkit> perl .\Tools\Scripts\build-webkit --release
CMake Error at CMakeLists.txt:6 (project):
  No CMAKE_C_COMPILER could be found.



CMake Error at CMakeLists.txt:6 (project):
  No CMAKE_CXX_COMPILER could be found.



Error: --system-information failed on internal CMake!
+  cmake -DPORT="Win" -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE="C:\MSVS\VC\vcpkg\\scripts\buildsystems\vcpkg.cmake" -DVCPKG_TARGET_TRIPLET=x64-windows-webkit -G Ninja -DDEVELOPER_MODE=ON -DENABLE_EXPERIMENTAL_FEATURES=ON -DVCPKG_MANIFEST_FEATURES=web;jpeg-xl;lcms;skia;woff2 "C:/webkit"
-- Running vcpkg install
Detecting compiler hash for triplet x64-windows...
Compiler found: C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe
Detecting compiler hash for triplet x64-windows-webkit...
Compiler found: C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe
The following packages are already installed:
  * vcpkg-cmake:x64-windows@2024-04-23 -- git+https://github.com/microsoft/vcpkg@e74aa1e8f93278a8e71372f1fa08c3df420eb840
  * vcpkg-cmake-config:x64-windows@2024-05-23 -- git+https://github.com/microsoft/vcpkg@97a63e4bc1a17422ffe4eff71da53b4b561a7841
The following packages will be built and installed:
    brotli:x64-windows-webkit@1.1.0#1 -- git+https://github.com/microsoft/vcpkg@4e5b5ae1ad26c80535c893cc0307121f0398549e
    curl[core,http3,ipv6,libressl]:x64-windows-webkit@8.13.0#1 -- git+https://github.com/WebKitForWindows/WebKitRequirements@22da8f318769e781d052f3e07db6c81007b85eb1
    harfbuzz[core,directwrite,icu]:x64-windows-webkit@11.3.3 -- git+https://github.com/microsoft/vcpkg@14f2e7630c576d41e7ae856b844423cb4a14b87a
  * highway:x64-windows-webkit@1.3.0 -- git+https://github.com/microsoft/vcpkg@5f81b96695f9e09191745ddb2308fb98ac33164c
    icu:x64-windows-webkit@77.1.0 -- git+https://github.com/WebKitForWindows/WebKitRequirements@ea40f10ddf278b30180b541393a9045baebe740c
    lcms:x64-windows-webkit@2.17 -- git+https://github.com/microsoft/vcpkg@e2e326389c873aa7ade1f31337e5df8627147147
    libjpeg-turbo:x64-windows-webkit@3.1.2 -- git+https://github.com/microsoft/vcpkg@26b02f049c1cbfc9d996c7b56cfd1f4598f6f158
    libjxl:x64-windows-webkit@0.11.1#4 -- git+https://github.com/microsoft/vcpkg@ac044181841dbaf63e6403aa3931b944574b2bdf
    libpng:x64-windows-webkit@1.6.50 -- git+https://github.com/microsoft/vcpkg@54b6eefbef1cb0f477c6eb54e2236badd140feea
    libpsl[core,libicu]:x64-windows-webkit@0.21.5#1 -- git+https://github.com/microsoft/vcpkg@1f1ba362bcb9a330a5882ff668eb7fc3a2c2b6c3
    libressl:x64-windows-webkit@4.2.0 -- git+https://github.com/microsoft/vcpkg@29d9b59ae2e3ab8f916d806eccf19f26253312ce
    libwebp[core,libwebpmux,nearlossless,simd,unicode]:x64-windows-webkit@1.6.0#1 -- git+https://github.com/microsoft/vcpkg@c0a4a598eacd380393ff3a92be045f4474643649
    libxml2[core,icu]:x64-windows-webkit@2.15.0 -- git+https://github.com/microsoft/vcpkg@8d4f6f9936bc885f1acb93fde7584b6fbe32e150
    libxslt[core,profiler,thread]:x64-windows-webkit@1.1.43#1 -- git+https://github.com/microsoft/vcpkg@e1f6fcb93d8c22da682e05de3b458ce33679976e
    nghttp2:x64-windows-webkit@1.67.1 -- git+https://github.com/microsoft/vcpkg@b7acda5edbc8182345cc0ec213d27444f7576f62
    nghttp3:x64-windows-webkit@1.12.0 -- git+https://github.com/microsoft/vcpkg@a4dc2dbdf993123d453152dddf62657f23bae85b
    ngtcp2[core,openssl]:x64-windows-webkit@1.12.0 -- git+https://github.com/microsoft/vcpkg@504d33e51bfe97e033e171a19c4994d93f11c9e9
  * openssl:x64-windows-webkit@libressl -- git+https://github.com/WebKitForWindows/WebKitRequirements@49722904b3d436ac499fc122102558b10ee73be1
    sqlite3[core,fts3,json1,rtree]:x64-windows-webkit@3.50.4 -- git+https://github.com/microsoft/vcpkg@66b97ff5a04f74e09b321708432721c5dca0c213
  * vcpkg-cmake-get-vars:x64-windows@2025-05-29 -- git+https://github.com/microsoft/vcpkg@d6510f888dd526c7828f6b973349e7f30be51254
  * vcpkg-tool-meson:x64-windows@1.9.0#2 -- git+https://github.com/microsoft/vcpkg@e367b6c164136bca2cb53f0b8308932f0ccd33b9
    woff2:x64-windows-webkit@1.0.2#5 -- git+https://github.com/microsoft/vcpkg@5a12f112e02313670c8e24d72d72a83ce74521b4
    zlib:x64-windows-webkit@zlib-ng -- git+https://github.com/WebKitForWindows/WebKitRequirements@b7e95e97d224d1a520ec9d5246f76c1e52bb4c8c
  * zlib-ng:x64-windows-webkit@2.2.5 -- git+https://github.com/microsoft/vcpkg@0e5ef520a2162b3f1864df8e803e7fb2e5da1d90
Additional packages (*) will be modified to complete this operation.
Restored 0 package(s) from C:\Users\ContainerAdministrator\AppData\Local\vcpkg\archives in 324 us. Use --debug to see more details.
Installing 1/24 icu:x64-windows-webkit@77.1.0...
Building icu:x64-windows-webkit@77.1.0...
C:\webkit\WebKitLibraries\triplets\x64-windows-webkit.cmake: info: loaded overlay triplet from here
C:\Users\ContainerAdministrator\AppData\Local\vcpkg\registries\git-trees\ea40f10ddf278b30180b541393a9045baebe740c: info: installing from git registry git+https://github.com/WebKitForWindows/WebKitRequirements@ea40f10ddf278b30180b541393a9045baebe740c
-- Using cached icu4c-77_1-src.tgz
-- Cleaning sources at C:/webkit/vcpkg_installed/vcpkg/blds/icu/src/c-77_1-src-e4739f0a52.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Users/ContainerAdministrator/AppData/Local/vcpkg/downloads/icu4c-77_1-src.tgz
-- Applying patch C:/Users/ContainerAdministrator/AppData/Local/vcpkg/registries/git-trees/ea40f10ddf278b30180b541393a9045baebe740c/patches/0001-Add-CMake-platform.patch
-- Applying patch C:/Users/ContainerAdministrator/AppData/Local/vcpkg/registries/git-trees/ea40f10ddf278b30180b541393a9045baebe740c/patches/0002-Remove-install-suffix-on-Windows.patch
-- Using source at C:/webkit/vcpkg_installed/vcpkg/blds/icu/src/c-77_1-src-e4739f0a52.clean
-- Configuring x64-windows-webkit
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: C:/Users/ContainerAdministrator/AppData/Local/vcpkg/downloads/tools/ninja/1.12.1-windows/ninja.exe -v
    Working Directory: C:/webkit/vcpkg_installed/vcpkg/blds/icu/x64-windows-webkit-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\webkit\vcpkg_installed\vcpkg\blds\icu\config-x64-windows-webkit-dbg-CMakeCache.txt.log
      C:\webkit\vcpkg_installed\vcpkg\blds\icu\config-x64-windows-webkit-rel-CMakeCache.txt.log
      C:\webkit\vcpkg_installed\vcpkg\blds\icu\config-x64-windows-webkit-dbg-CMakeConfigureLog.yaml.log
      C:\webkit\vcpkg_installed\vcpkg\blds\icu\config-x64-windows-webkit-rel-CMakeConfigureLog.yaml.log
      C:\webkit\vcpkg_installed\vcpkg\blds\icu\config-x64-windows-webkit-out.log

Call Stack (most recent call first):
  C:/webkit/vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:269 (vcpkg_execute_required_process)
  C:/Users/ContainerAdministrator/AppData/Local/vcpkg/registries/git-trees/ea40f10ddf278b30180b541393a9045baebe740c/portfile.cmake:54 (vcpkg_cmake_configure)
  scripts/ports.cmake:206 (include)


error: building icu:x64-windows-webkit failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle icu:x64-windows-webkit: 4.7 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+icu
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=%5Bicu%5D%20build%20error%20on%20x64-windows-webkit&body=Copy%20issue%20body%20from%20C%3A%2Fwebkit%2Fvcpkg_installed%2Fvcpkg%2Fissue_body.md

-- Running vcpkg install - failed
CMake Error at C:/MSVS/VC/vcpkg/scripts/buildsystems/vcpkg.cmake:941 (message):
  vcpkg install failed.  See logs for more information:
  C:\webkit\vcpkg-manifest-install.log
Call Stack (most recent call first):
  C:/tools/cmake/share/cmake-3.31/Modules/CMakeDetermineSystem.cmake:146 (include)
  CMakeLists.txt:10 (project)


CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
Run Visual Studio installation vcvars.bat before build-webkit when using ninja at .\Tools\Scripts\build-webkit line 293.
</pre>
</details> 

<details><summary>vcpkg_installed\vcpkg\blds\icu\config-x64-windows-webkit-out.log</summary>
<pre>
[1/2] "C:/tools/cmake/bin/cmake.exe" -E chdir ".." "C:/tools/cmake/bin/cmake.exe" "C:/webkit/vcpkg_installed/vcpkg/blds/icu/src/c-77_1-src-e4739f0a52.clean/source" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/webkit/vcpkg_installed/vcpkg/pkgs/icu_x64-windows-webkit" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DICU_ENABLE_PLUGINS=OFF" "-DICU_ENABLE_EXTRAS=OFF" "-DICU_ENABLE_SAMPLES=OFF" "-DICU_ENABLE_TESTS=OFF" "-DICU_ENABLE_TOOLS=ON" "-DICU_CROSS_COMPILING=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Users/ContainerAdministrator/AppData/Local/vcpkg/downloads/tools/ninja/1.12.1-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/MSVS/VC/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-webkit" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/MSVS/VC/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/MSVS/VC/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/webkit/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../CMakeCache.txt 
"C:/tools/cmake/bin/cmake.exe" -E chdir ".." "C:/tools/cmake/bin/cmake.exe" "C:/webkit/vcpkg_installed/vcpkg/blds/icu/src/c-77_1-src-e4739f0a52.clean/source" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/webkit/vcpkg_installed/vcpkg/pkgs/icu_x64-windows-webkit" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DICU_ENABLE_PLUGINS=OFF" "-DICU_ENABLE_EXTRAS=OFF" "-DICU_ENABLE_SAMPLES=OFF" "-DICU_ENABLE_TESTS=OFF" "-DICU_ENABLE_TOOLS=ON" "-DICU_CROSS_COMPILING=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Users/ContainerAdministrator/AppData/Local/vcpkg/downloads/tools/ninja/1.12.1-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/MSVS/VC/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-webkit" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/MSVS/VC/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/MSVS/VC/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/webkit/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
-- for ICU version numbers
-- release 77.1, library 77.1, unicode version 16.0
-- The C compiler identification is MSVC 19.44.35217.0
-- The CXX compiler identification is MSVC 19.44.35217.0
-- The ASM compiler identification is MSVC
-- Found assembler: C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - failed
-- Check for working C compiler: C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe
-- Check for working C compiler: C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe - broken
CMake Error at C:/tools/cmake/share/cmake-3.31/Modules/CMakeTestCCompiler.cmake:67 (message):
  The C compiler

    "C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: 'C:/webkit/vcpkg_installed/vcpkg/blds/icu/x64-windows-webkit-rel/CMakeFiles/CMakeScratch/TryCompile-d4wgjv'
    
    Run Build Command(s): C:/Users/ContainerAdministrator/AppData/Local/vcpkg/downloads/tools/ninja/1.12.1-windows/ninja.exe -v cmTC_fbeda
    [1/2] C:\MSVS\VC\Tools\MSVC\14.44.35207\bin\Hostx64\x64\cl.exe  /nologo   /nologo /DWIN32 /D_WINDOWS /utf-8 /MP   /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_fbeda.dir\testCCompiler.c.obj /FdCMakeFiles\cmTC_fbeda.dir\ /FS -c C:\webkit\vcpkg_installed\vcpkg\blds\icu\x64-windows-webkit-rel\CMakeFiles\CMakeScratch\TryCompile-d4wgjv\testCCompiler.c
    [2/2] C:\Windows\system32\cmd.exe /C "cd . && C:\tools\cmake\bin\cmake.exe -E vs_link_exe --msvc-ver=1944 --intdir=CMakeFiles\cmTC_fbeda.dir --rc="C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\rc.exe" --mt="C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\mt.exe" --manifests  -- C:\MSVS\VC\Tools\MSVC\14.44.35207\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_fbeda.dir\testCCompiler.c.obj  /out:cmTC_fbeda.exe /implib:cmTC_fbeda.lib /pdb:cmTC_fbeda.pdb /version:0.0 /machine:x64  /nologo    /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
    FAILED: cmTC_fbeda.exe 
    C:\Windows\system32\cmd.exe /C "cd . && C:\tools\cmake\bin\cmake.exe -E vs_link_exe --msvc-ver=1944 --intdir=CMakeFiles\cmTC_fbeda.dir --rc="C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\rc.exe" --mt="C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\mt.exe" --manifests  -- C:\MSVS\VC\Tools\MSVC\14.44.35207\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_fbeda.dir\testCCompiler.c.obj  /out:cmTC_fbeda.exe /implib:cmTC_fbeda.lib /pdb:cmTC_fbeda.pdb /version:0.0 /machine:x64  /nologo    /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
    RC Pass 1: command "C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\rc.exe /fo CMakeFiles\cmTC_fbeda.dir/manifest.res CMakeFiles\cmTC_fbeda.dir/manifest.rc" failed (exit code 0) with the following output:

    Access violation

    ninja: build stopped: subcommand failed.

    
    

  

  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:39 (project)


-- Configuring incomplete, errors occurred!
[2/2] "C:/tools/cmake/bin/cmake.exe" -E chdir "../../x64-windows-webkit-dbg" "C:/tools/cmake/bin/cmake.exe" "C:/webkit/vcpkg_installed/vcpkg/blds/icu/src/c-77_1-src-e4739f0a52.clean/source" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/webkit/vcpkg_installed/vcpkg/pkgs/icu_x64-windows-webkit/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DICU_ENABLE_PLUGINS=OFF" "-DICU_ENABLE_EXTRAS=OFF" "-DICU_ENABLE_SAMPLES=OFF" "-DICU_ENABLE_TESTS=OFF" "-DICU_ENABLE_TOOLS=ON" "-DICU_CROSS_COMPILING=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Users/ContainerAdministrator/AppData/Local/vcpkg/downloads/tools/ninja/1.12.1-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/MSVS/VC/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-webkit" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/MSVS/VC/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/MSVS/VC/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/webkit/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../../x64-windows-webkit-dbg/CMakeCache.txt 
"C:/tools/cmake/bin/cmake.exe" -E chdir "../../x64-windows-webkit-dbg" "C:/tools/cmake/bin/cmake.exe" "C:/webkit/vcpkg_installed/vcpkg/blds/icu/src/c-77_1-src-e4739f0a52.clean/source" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/webkit/vcpkg_installed/vcpkg/pkgs/icu_x64-windows-webkit/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DICU_ENABLE_PLUGINS=OFF" "-DICU_ENABLE_EXTRAS=OFF" "-DICU_ENABLE_SAMPLES=OFF" "-DICU_ENABLE_TESTS=OFF" "-DICU_ENABLE_TOOLS=ON" "-DICU_CROSS_COMPILING=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Users/ContainerAdministrator/AppData/Local/vcpkg/downloads/tools/ninja/1.12.1-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/MSVS/VC/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-webkit" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/MSVS/VC/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/MSVS/VC/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/webkit/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
-- for ICU version numbers
-- release 77.1, library 77.1, unicode version 16.0
-- The C compiler identification is MSVC 19.44.35217.0
-- The CXX compiler identification is MSVC 19.44.35217.0
-- The ASM compiler identification is MSVC
-- Found assembler: C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - failed
-- Check for working C compiler: C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe
-- Check for working C compiler: C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe - broken
CMake Error at C:/tools/cmake/share/cmake-3.31/Modules/CMakeTestCCompiler.cmake:67 (message):
  The C compiler

    "C:/MSVS/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: 'C:/webkit/vcpkg_installed/vcpkg/blds/icu/x64-windows-webkit-dbg/CMakeFiles/CMakeScratch/TryCompile-t9dm10'
    
    Run Build Command(s): C:/Users/ContainerAdministrator/AppData/Local/vcpkg/downloads/tools/ninja/1.12.1-windows/ninja.exe -v cmTC_c0a84
    [1/2] C:\MSVS\VC\Tools\MSVC\14.44.35207\bin\Hostx64\x64\cl.exe  /nologo   /nologo /DWIN32 /D_WINDOWS /utf-8 /MP   /MDd /Z7 /Ob0 /Od /RTC1 /showIncludes /FoCMakeFiles\cmTC_c0a84.dir\testCCompiler.c.obj /FdCMakeFiles\cmTC_c0a84.dir\ /FS -c C:\webkit\vcpkg_installed\vcpkg\blds\icu\x64-windows-webkit-dbg\CMakeFiles\CMakeScratch\TryCompile-t9dm10\testCCompiler.c
    [2/2] C:\Windows\system32\cmd.exe /C "cd . && C:\tools\cmake\bin\cmake.exe -E vs_link_exe --msvc-ver=1944 --intdir=CMakeFiles\cmTC_c0a84.dir --rc="C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\rc.exe" --mt="C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\mt.exe" --manifests  -- C:\MSVS\VC\Tools\MSVC\14.44.35207\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_c0a84.dir\testCCompiler.c.obj  /out:cmTC_c0a84.exe /implib:cmTC_c0a84.lib /pdb:cmTC_c0a84.pdb /version:0.0 /machine:x64  /nologo    /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
    FAILED: cmTC_c0a84.exe 
    C:\Windows\system32\cmd.exe /C "cd . && C:\tools\cmake\bin\cmake.exe -E vs_link_exe --msvc-ver=1944 --intdir=CMakeFiles\cmTC_c0a84.dir --rc="C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\rc.exe" --mt="C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\mt.exe" --manifests  -- C:\MSVS\VC\Tools\MSVC\14.44.35207\bin\Hostx64\x64\link.exe /nologo CMakeFiles\cmTC_c0a84.dir\testCCompiler.c.obj  /out:cmTC_c0a84.exe /implib:cmTC_c0a84.lib /pdb:cmTC_c0a84.pdb /version:0.0 /machine:x64  /nologo    /debug /INCREMENTAL /subsystem:console  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."

    RC Pass 1: command "C:\Program Files (x86)\Windows Kits\10\bin\10.0.26100.0\x64\rc.exe /fo CMakeFiles\cmTC_c0a84.dir/manifest.res CMakeFiles\cmTC_c0a84.dir/manifest.rc" failed (exit code 0) with the following output:

    Access violation

    ninja: build stopped: subcommand failed.

    
    

  

  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:39 (project)


-- Configuring incomplete, errors occurred!
ninja: build stopped: subcommand failed.
</pre>
</details>

--- a/.github/workflows/docker-image.yml
+++ b/.github/workflows/docker-image.yml
@@ -16,9 +16,44 @@ jobs:
           os: windows-2022
         - tag: windows-2022
           os: windows-2022
+        - tag: 2025
+          os: windows-2025
+        - tag: windows-2025
+          os: windows-2025
     runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v4
+
+    # Free disk space by removing pre-installed software we don't need
+    - name: Free disk space
+      run: |
+        Write-Host "Disk space before cleanup:"
+        Get-PSDrive C | Select-Object Used, Free | Format-Table
+        $paths = @(
+          'C:\Android',
+          'C:\Program Files\dotnet',
+          'C:\Program Files\MongoDB',
+          'C:\Program Files\PostgreSQL',
+          'C:\Selenium',
+          'C:\hostedtoolcache',
+          'C:\Program Files\Microsoft Visual Studio',
+          'C:\vcpkg',
+          'C:\Miniconda',
+          'C:\Program Files\Go',
+          'C:\Program Files\Node.js',
+          'C:\Program Files\Ruby',
+          'C:\Program Files\Java',
+          'C:\Program Files\Eclipse Adoptium'
+        )
+        foreach ($path in $paths) {
+          if (Test-Path $path) {
+            Write-Host "Removing $path ..."
+            Remove-Item -Recurse -Force $path -ErrorAction SilentlyContinue
+          }
+        }
+        docker system prune -a -f
+        Write-Host "Disk space after cleanup:"
+        Get-PSDrive C | Select-Object Used, Free | Format-Table
 
     # Build Docker images
     - name: Build webkitdev/base
@@ -26,7 +61,7 @@ jobs:
     - name: Build webkitdev/scripts
       run: docker build --tag webkitdev/scripts:${{ matrix.tag }} --build-arg IMAGE_TAG=${{ matrix.tag }} scripts
     - name: Build webkitdev/scm
-      run: docker build --tag webkitdev/scm:${{ matrix.tag }} --build-arg IMAGE_TAG=${{ matrix.tag }} scm      
+      run: docker build --tag webkitdev/scm:${{ matrix.tag }} --build-arg IMAGE_TAG=${{ matrix.tag }} scm
     - name: Build webkitdev/tools
       run: docker build --tag webkitdev/tools:${{ matrix.tag }} --build-arg IMAGE_TAG=${{ matrix.tag }} tools
     - name: Build webkitdev/msbuild-2022
@@ -38,7 +73,7 @@ jobs:
     - name: Verify webkitdev/buildbot-worker
       run: docker run --rm webkitdev/buildbot-worker:${{ matrix.tag }} cmake --system-information
 
-    # Publish Docker images
+    # Publish Docker images to DockerHub
     - name: Dockerhub login
       if: github.ref_name == github.event.repository.default_branch
       uses: docker/login-action@v3

--- a/BUILDBOT.md
+++ b/BUILDBOT.md
@@ -3,7 +3,7 @@ The `docker-webkit-dev` project contains Docker images for running Buildbot
 workers that connect to WebKit infrastructure. The Windows WebKit port builds
 happen within Docker containers running the `webkitdev/buildbot-worker` image.
 
-> [!IMPORTANT]  
+> [!IMPORTANT]
 > To connect to WebKit infrastructure credentials are required.
 > Contact the infrastructure team on [Slack](https://webkit.slack.com) by
 > dropping a message in the `#dev` channel for assistance in connecting new
@@ -18,9 +18,10 @@ compatibility see the
 
 | Tag Name  | Automated | Description |
 |---|:---:|---|
-| 2022 | :white_check_mark: | A Windows 2022 server container |
+| 2022 | :white_check_mark: | A Windows Server 2022 container |
 | windows-2022 | :x: | A Windows container, used for Layout Tests |
-| 1809 | :x: | A Windows 2019 server container |
+| 2025 | :white_check_mark: | A Windows Server 2025 container |
+| windows-2025 | :x: | A Windows container, used for Layout Tests |
 
 Visit [Docker Hub](https://hub.docker.com/r/webkitdev/buildbot-worker/tags) to
 see when the last build happened for the image's tags.
@@ -39,7 +40,7 @@ for more details.
 | BUILD_WORKER_NAME | :white_check_mark: | Name for the worker, provided by infrastructure team  |
 | BUILD_WORKER_PASSWORD | :white_check_mark: | Password for the worker, provided by infrastructure team |
 | BUILD_WORKER_KEEPALIVE | :x: | Time in seconds that messages should be sent by the worker to the server, defaults to 240 |
-| ADMIN_NAME | :x:| Contact name for the admin of the worker | 
+| ADMIN_NAME | :x:| Contact name for the admin of the worker |
 | ADMIN_EMAIL | :x: | Contact e-mail address for the admin of the worker |
 | HOST_DESCRIPTION |:x: | A description of the host running the worker |
 | COMPILER | :white_check_mark: | Needs to be set to `Clang` since the alternative, `cl`, is no longer able to build WebKit |
@@ -60,8 +61,8 @@ COMPILER=Clang
 ```
 
 ## Running the Buildbot worker
-On a Windows Server 2022 machine run the following in a powershell session. On
-Windows Server 2022 the Docker container runs in process isolation mode which
+On a Windows Server 2022/2025 machine run the following in a powershell session.
+On Windows Server 2022/2025 the Docker container runs in process isolation mode which
 gives it access to all the resources of the host.
 
 > [!NOTE]

--- a/Build-All.ps1
+++ b/Build-All.ps1
@@ -1,6 +1,6 @@
 param(
   [Parameter(Mandatory)]
-  [ValidateSet('1809','2022','aws','windows-1809','windows-aws','windows-2022')]
+  [ValidateSet('2022','windows-2022','2025','windows-2025')]
   [string]$tag
 )
 

--- a/Publish-All.ps1
+++ b/Publish-All.ps1
@@ -1,6 +1,6 @@
 param(
   [Parameter(Mandatory)]
-  [ValidateSet('1809','2022','aws','windows-1809','windows-aws','windows-2022')]
+  [ValidateSet('2022','windows-2022','2025','windows-2025')]
   [string]$tag
 )
 

--- a/README.md
+++ b/README.md
@@ -5,9 +5,9 @@ Docker images for local WebKit development and CI/CD on Windows.
 
 ## Host Setup
 Using the `webkitdev` Docker images requires a Windows host, ideally Windows 11
-or Windows Server 2022 but Windows 10 and Windows Server 2019 can also be used,
-with [Docker](https://www.docker.com/) installed and targeting Windows
-containers. For new installs follow the latest documentation to setup
+or Windows Server 2022/2025, with [Docker](https://www.docker.com/) installed
+and targeting Windows containers. For new installs follow the latest
+documentation to setup
 [Windows for containers](https://learn.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment).
 
 ## Images
@@ -29,18 +29,16 @@ WebKit infrastructure.
 | [buildbot-worker](https://hub.docker.com/r/webkitdev/buildbot-worker) | Contains Buildbot and scripts to connect to WebKit CI/CD infrastructure |
 
 Docker images support tagging. For Windows images the tag references the version
-of the container base image, Windows Server 2022 and Windows Server 2019.
-Compatibility depends on what the host OS is. In general Windows 11 needs to
-target 2022 tags and Windows 10 needs to target 2019 tags. For the latest
-information on Windows container version compatibility see the
+of the container base image. Compatibility depends on what the host OS is. For
+the latest information on Windows container version compatibility see the
 [documentation](https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility).
 
-| Tag | Automated | Win 11 | Win 10 | Description |
-|---|:---:|:---:|:---:|---|
-| 2022 | :white_check_mark: | :white_check_mark: | :x: | A Windows 2022 server container |
-| windows-2022 | :x: | :white_check_mark: | :x: | A Windows container, used for Layout Tests |
-| 1809 | :x: | :white_check_mark: | :white_check_mark: | A Windows 2019 server container |
-| windows-1809 | :x: | :white_check_mark: | :white_check_mark: | A Windows container, used for Layout Tests |
+| Tag | Automated | Description |
+|---|:---:|---|
+| 2022 | :white_check_mark: | A Windows Server 2022 container |
+| windows-2022 | :x: | A Windows container, used for Layout Tests |
+| 2025 | :white_check_mark: | A Windows Server 2025 container |
+| windows-2025 | :x: | A Windows container, used for Layout Tests |
 
 The `windows-<version>` have a larger base image containing more Windows OS
 components making them ideal for testing WebKit. The other tags use Windows
@@ -48,7 +46,7 @@ Server Core and are suitable for building WebKit.
 
 ### Building locally
 > [!IMPORTANT]
-> Windows 11 and Windows Server 2022 users should pull the images directly from
+> Windows 11 and Windows Server 2022/2025 users should pull the images directly from
 > DockerHub rather than building locally. The only exception is when
 > [updating the images](UPDATING.md).
 
@@ -61,7 +59,7 @@ and tagged. The created time should be within the time frame the script was
 executing in.
 
 ## Building the Windows WebKit port
-With the `webkitdev/msbuild-2022` image everything is there to do a build of 
+With the `webkitdev/msbuild-2022` image everything is there to do a build of
 the Windows WebKit port. Start out by doing a local checkout of the
 [WebKit repository](https://github.com/WebKit/WebKit). The `docker run` command
 needs to be populated with the following fields.
@@ -70,7 +68,7 @@ needs to be populated with the following fields.
 |---|---|
 | tag | The tag to use |
 | cpu-count | The number of CPUs to dedicate to the container (optional on a Windows Server host) |
-| [memory](https://docs.docker.com/reference/cli/docker/container/run/#memory) | The memory limit for the container (optional on a Windows Server host) | 
+| [memory](https://docs.docker.com/reference/cli/docker/container/run/#memory) | The memory limit for the container (optional on a Windows Server host) |
 | [volume](https://docs.docker.com/reference/cli/docker/container/run/#volume) | A local path containing the WebKit checkout, use `/` over `\` |
 
 ```powershell
@@ -88,7 +86,7 @@ like this :point_down:.
 docker run --name build --rm -it `
     --cpu-count=6 --memory=16g `
     --volume C:/GitHub/webkit:C:/webkit `
-    webkitdev/msbuild-2022:2022 powershell
+    webkitdev/msbuild-2022:2025 powershell
 ```
 
 Once the command is run it will place you into a Powershell session. From there
@@ -103,9 +101,9 @@ perl Tools\Scripts\build-webkit
 ```
 
 > [!NOTE]
-> Building in a container in Hyper-V isolation, the default for Windows 11 and
-> 10, will take longer than a local build. Building in a container in process
-> mode, the default for Windows Server 2022 and 2019, will build in a similar
+> Building in a container in Hyper-V isolation, the default for Windows 11,
+> will take longer than a local build. Building in a container in process
+> mode, the default for Windows Server 2022 and 2025, will build in a similar
 > time as a local build.
 >
 > Ideally dedicate a large amount of resources when running in Hyper-V to reduce

--- a/UPDATING.md
+++ b/UPDATING.md
@@ -45,7 +45,7 @@ Use the `Build-All.ps1` PowerShell script to build the images. Again it expects
 a single argument `-Tag` which specifies the tag to build.
 
 ```powershell
-./Build-All -tag 2022
+./Build-All -tag 2025
 ```
 
 Executing the script will create all the `webkitdev` images. Most of `RUN`

--- a/base/Dockerfile.1809
+++ b/base/Dockerfile.1809
@@ -1,1 +1,0 @@
-FROM mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2019@sha256:1d05469f3f6d595f05531b4f58157b421526162b26c97a19d98dfd08a5584924

--- a/base/Dockerfile.2025
+++ b/base/Dockerfile.2025
@@ -1,0 +1,1 @@
+FROM mcr.microsoft.com/dotnet/framework/runtime:4.8.1-windowsservercore-ltsc2025@sha256:fdb22792bfb201f78ba84c54dc76e9f062bb811b9441bfc5c70ec6dd9b5c4c95

--- a/base/Dockerfile.aws
+++ b/base/Dockerfile.aws
@@ -1,1 +1,0 @@
-FROM mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2019@sha256:578d07650a3dfe8db5b988f26aa0c23180a248bcb7f5e1ab1d7ba7713b73ad43

--- a/base/Dockerfile.windows-2025
+++ b/base/Dockerfile.windows-2025
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/windows:1809@sha256:5dc542418c860e622a8da5d2db2dd3f3eb1212c104857ca947dac2fd171a26b5
+FROM mcr.microsoft.com/windows/server:ltsc2025@sha256:bfb0496d2ff297e49b581ea16166ec955d4afbe94a412ad99db52ce554fb327c
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 

--- a/base/Dockerfile.windows-aws
+++ b/base/Dockerfile.windows-aws
@@ -1,6 +1,0 @@
-FROM mcr.microsoft.com/windows:1809@sha256:bf5f60839adcb5cfd775243bb86069deb88083217c656c456d90fbb685369aa5
-
-SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
-
-RUN Set-ExecutionPolicy -ExecutionPolicy RemoteSigned;
-

--- a/base/manifest.tmpl
+++ b/base/manifest.tmpl
@@ -7,14 +7,14 @@ tags:
 {{/if}}
 manifests:
   -
-    image: webkitdev/base:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}1809
-    platform:
-      architecture: amd64
-      os: windows
-      version: 1809
-  -
     image: webkitdev/base:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2022
     platform:
       architecture: amd64
       os: windows
       version: 2022
+  -
+    image: webkitdev/base:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2025
+    platform:
+      architecture: amd64
+      os: windows
+      version: 2025

--- a/buildbot-worker/manifest.tmpl
+++ b/buildbot-worker/manifest.tmpl
@@ -7,14 +7,14 @@ tags:
 {{/if}}
 manifests:
   -
-    image: webkitdev/buildbot-worker{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}1809
-    platform:
-      architecture: amd64
-      os: windows
-      version: 1809
-  -
     image: webkitdev/buildbot-worker{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2022
     platform:
       architecture: amd64
       os: windows
       version: 2022
+  -
+    image: webkitdev/buildbot-worker{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2025
+    platform:
+      architecture: amd64
+      os: windows
+      version: 2025

--- a/msbuild-2022/manifest.tmpl
+++ b/msbuild-2022/manifest.tmpl
@@ -7,14 +7,14 @@ tags:
 {{/if}}
 manifests:
   -
-    image: webkitdev/msbuild-2022:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}1809
-    platform:
-      architecture: amd64
-      os: windows
-      version: 1809
-  -
     image: webkitdev/msbuild-2022:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2022
     platform:
       architecture: amd64
       os: windows
       version: 2022
+  -
+    image: webkitdev/msbuild-2022:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2025
+    platform:
+      architecture: amd64
+      os: windows
+      version: 2025

--- a/renovate.json
+++ b/renovate.json
@@ -6,8 +6,6 @@
   "enabledManagers": ["dockerfile"],
   "dockerfile": {
     "ignorePaths": [
-      "base/Dockerfile.aws",
-      "base/Dockerfile.windows-aws",
       "scripts/",
       "scm/",
       "tools/",

--- a/scm/manifest.tmpl
+++ b/scm/manifest.tmpl
@@ -7,14 +7,14 @@ tags:
 {{/if}}
 manifests:
   -
-    image: webkitdev/scm:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}1809
-    platform:
-      architecture: amd64
-      os: windows
-      version: 1809
-  -
     image: webkitdev/scm:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2022
     platform:
       architecture: amd64
       os: windows
       version: 2022
+  -
+    image: webkitdev/scm:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2025
+    platform:
+      architecture: amd64
+      os: windows
+      version: 2025

--- a/scripts/manifest.tmpl
+++ b/scripts/manifest.tmpl
@@ -7,14 +7,14 @@ tags:
 {{/if}}
 manifests:
   -
-    image: webkitdev/scripts:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}1809
-    platform:
-      architecture: amd64
-      os: windows
-      version: 1809
-  -
     image: webkitdev/scripts:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2022
     platform:
       architecture: amd64
       os: windows
-      version: 20H2
+      version: 2022
+  -
+    image: webkitdev/scripts:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2025
+    platform:
+      architecture: amd64
+      os: windows
+      version: 2025

--- a/tools/manifest.tmpl
+++ b/tools/manifest.tmpl
@@ -7,14 +7,14 @@ tags:
 {{/if}}
 manifests:
   -
-    image: webkitdev/tools:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}1809
-    platform:
-      architecture: amd64
-      os: windows
-      version: 1809
-  -
     image: webkitdev/tools:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2022
     platform:
       architecture: amd64
       os: windows
       version: 2022
+  -
+    image: webkitdev/tools:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}2025
+    platform:
+      architecture: amd64
+      os: windows
+      version: 2025

